bftengine/src/bcstatetransfer/AsyncStateTransferCRE.cpp

```diff
@@ -136,8 +129,41 @@ class ScalingReplicaHandler : public IStateHandler {
     return logger_;
   }
 };
-std::shared_ptr<ClientReconfigurationEngine> CreFactory::create(std::shared_ptr<MsgsCommunicator> msgsCommunicator,
-                                                                std::shared_ptr<MsgHandlersRegistrator> msgHandlers) {
+
+// TODO(yf): remove
```

Comment:
remove the TODO, this handler is used by the read-only replica

Reply:
Done